app/controllers/licences.controller.js

```diff
@@ -6,8 +6,8 @@
  */
 
 const InitiateReturnRequirementSessionService = require('../services/return-requirements/initiate-return-requirement-session.service.js')
-const ViewLicenceSummaryService = require('../services/licences/view-license-summary.service')
-
+const ViewLicenceSummaryService = require('../services/licences/view-licence-summary.service')
+const ViewLicenceReturnsService = require('../services/licences/view-licence-returns.service')
```

Comment:
NIT - alphabetical ordering and adding some white space

```suggestion
const ViewLicenceReturnsService = require('../services/licences/view-licence-returns.service')
const ViewLicenceSummaryService = require('../services/licences/view-licence-summary.service')

```